spec/project-spec.coffee

```diff
@@ -436,6 +436,29 @@ describe "Project", ->
       expect(atom.project.getPaths()).toEqual([path.join(__dirname, "..", "src")])
       expect(atom.project.getRepositories()[0].isSubmodule("src")).toBe false
 
+    it "removes a path that is represented as a URI", ->
+      ftpUri = "ftp://example.com/some/folder"
+      directoryProvider =
+        directoryForURISync: (uri) ->
+          if uri is ftpUri
```

Comment:
Sure thing: I wasn't sure how defensive you wanted the test to be. All cleaned up.